gst/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.c

```diff
@@ -40,6 +67,71 @@ struct _Tflite_data
 };
 typedef struct _Tflite_data tflite_data;
 
+/**
+  * @brief parse custom options for tensorflow lite
+  * @param filter : tensor_filter instance
+  * @param id : Option ID
+  * @return 0 or 1 if successfully parsed.
+  *        -1 if parsing a value of a given option failed.
+  */
+static int
+tflite_get_parser_boolean (const GstTensorFilter * filter, int id)
+{
+  int ret = -1;
+
+  if (filter->prop.custom_properties == NULL)
+    return ret;
+
+  if (id < 0 || id >= TFLITE_OPTION_MAX) {
+    GST_WARNING ("Invalid option id. (%d)\n", id);
+    return ret;
+  }
+
+  custom_options = g_strdup (filter->prop.custom_properties);
+  if (custom_options == NULL) {
+    GST_WARNING ("failed to dump custom property string\n");
+    return ret;
+  }
+
+  char *str = strtok (custom_options, tokens);
+  while (str != NULL) {
+    if (strcmp (str, tflite_options[id]) == 0) {
+      str = strtok (NULL, tokens);
+      break;
+    }
+
+    /** TODO. check other options here if required. */
+
+    str = strtok (NULL, tokens);
+    if (str == NULL)
+      break;
+
+    str = strtok (NULL, tokens);
+  }
+
+  if (str == NULL) {
+    GST_WARNING ("Invalid custom option.\n");
+    goto out;
+  }
+
+  switch (id) {
+    case TFLITE_OPTION_NNAPI:
+      if (strcmp (str, "TRUE") == 0)
```

Comment:
Making the capital strings, "TRUE" and "FALSE",  be only valid values for this property looks too strict to me. You need to find out how other elements in gstreamer handle those boolean-value properties using gst-inspect-1.0. For example, 
in the case of 'fakesink',
```bash
  silent              : Don't produce last_message events
                        flags: readable, writable, changeable in NULL, READY, PAUSED or PLAYING state
                        Boolean. Default: true
  dump                : Dump buffer contents to stdout
                        flags: readable, writable, changeable in NULL, READY, PAUSED or PLAYING state
                        Boolean. Default: false
  signal-handoffs     : Send a signal before unreffing the buffer
                        flags: readable, writable
                        Boolean. Default: false
```

See alse: http://manpages.ubuntu.com/manpages/trusty/man1/gst-launch-1.0.1.html.